<!-- launch file when using vector nav -->

<launch>
    <arg name="filename" default="vector_bags"/>

    <node pkg="recovery_supervisor" type="recovery_supervisor"
          name="recovery_supervisor" output="screen">

        <param name="max_trip_time_error_factor" value="1.25" />
        <param name="average_forward_velocity" value=".25" />
        <param name="bag_file_directory_prefix" type="str" value="$(find recovery_supervisor)/bags/$(arg filename)" />

        <remap from="global_plan" to="/move_base/GlobalPlanner/plan" />
        <remap from="move_base/status" to="move_base_navi/status" />
        <remap from="velocity" to="vector/navigation/cmd_vel" />
        <remap from="odom" to="vector/odometry/local_filtered" />
        <remap from="move_base/cancel" to="move_base_navi/cancel" />
        <remap from="cmd_vel" to="vector/teleop/cmd_vel" />
        <remap from="demo_path" to="points_to_path/path" />
        <remap from="recovery_supervisor/state_feature" to="/move_base/global_costmap/demo_layer/state_feature" />

    </node>

    <node pkg="recovery_supervisor" name="points_to_path" type="points_to_path"/>

    <include file="$(find nav_points)/launch/rail.launch"/>


</launch>

